### PR TITLE
[chef-provisioning-aws] aws_rds_instance 

### PR DESCRIPTION
**Pull request by piyushawasthi**
`aws_rds_instance` resource: Migrate from AWS SDK V1 to V2	

Signed-off-by: piyushawasthi <piyush.awasthi@msystechnologies.com>

### CODE REVIEW

**Review by Vasu1105 — COMMENTED** [review]
LGTM

File: lib/chef/resource/aws_rds_parameter_group.rb [hunk 1]
```diff
@@ -13,18 +13,18 @@ class Chef::Resource::AwsRdsParameterGroup < Chef::Provisioning::AWSDriver::AWSR
...
     marker = initial_request[:marker]
     parameters = initial_request[:parameters]
     while !marker.nil?
-      more_results = driver.rds.client.describe_db_parameters(db_parameter_group_name: name, max_records: 100, marker: marker)
+      more_results = driver.rds.describe_db_parameters(db_parameter_group_name: name, max_records: 100, marker: marker)
       parameters += more_results[:parameters]
       marker = more_results[:marker]
     end
-    object[:parameters] = parameters
+    # object[:parameters] = parameters
```
Comment (NimishaS) [inline lib/chef/resource/aws_rds_parameter_group.rb:27]: Please remove the commented code.

**Comment by NimishaS** [comment]
@piyushawasthi, seems like this PR can be discarded. Please refer https://github.com/MsysTechnologiesllc/chef-provisioning-aws/pull/25#issuecomment-331412402

**Review by harikesh-kolekar — APPROVED** [review]

**Comment by NimishaS** [comment]
Closing this PR as https://github.com/MsysTechnologiesllc/chef-provisioning-aws/pull/25 is merged.

**Pull request closed without merging**